### ISSUE TITLE
pm2-windows-service does not start after reboot

### ISSUE THREAD

**Issue by kmzjy110** [jon-hall/pm2-windows-service#58]
Hi,
I have installed pm2-windows-service on my Windows Server 2019 remote server machine. Everything's been working well, except that the service does not start after I reboot the machine if I do not sign in as a user. The service starts right after I sign in as a user. After that, signing out/signing in does not affect the service. I was wondering if anyone has seen problems similar to this and if anyone has a solution for it?

**Comment by bestickley** [comment]
I ran into this same issue, but what fixed it for me was specifying the full file path for `PM2_SERVICE_SCRIPTS`

**Comment by TDola** [comment]
I've been having this too now.

I was able to get it working using 2 different methods.

Method 1: Add recovery options in windows services to make it try the service 1 minute after the initial failure.  This is kind of a blunt way of fixing it.
Method 2:  I switched to this fork https://github.com/mendrix/pm2-windows-service and downgraded PM2 to 4.2.1 and it started working again.  The downgrading appears to be the important bit, but the fork is more updated than this version so I switched to it.
